src/node.js

```diff
@@ -388,17 +389,17 @@
       tickDone();
     }
 
+    function PObj(c) {
```

Comment:
Yeah. It's late and I've been out most the day. A better name would be an improvement. Any ideas? :)